plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -300,3 +301,12 @@ func validatePriorityOrDie(priority schedulerapi.PriorityPolicy) {
 		}
 	}
 }
+
+// This function is called when listing all availabe algortihm providers in `kube-scheduler --help`
```

Comment:
Replace 'This function' with 'ListAlgorithmProviders' to conform with the godoc spec.